### ISSUE TITLE
Issues with Split Handler

### ISSUE THREAD

**Issue by crockk** [markusahlstrand/cloudworker-proxy#85]
Hi, this might be more of a support request than anything else, but I'm struggling to get the split handler to work with my Cloudflare worker. I'm just trying to get my worker to forward incoming requests to the listed host(s). Here's my `index.js`:
```
const Proxy = require('cloudworker-proxy');

config = [{
    handlerName: 'split',
    options: {
        host: 'public.server:9500/hooks/test'
    },
}];

const proxy = new Proxy(config);

async function fetchAndApply(event) {
    return await proxy.resolve(event);
}

addEventListener('fetch', (event) => {
  event.respondWith(fetchAndApply(event));
});
```
I can see the requests going through to the worker, and sometimes the worker logs a few subrequests, however the requests aren't hitting my `public.server`. Just hoping someone could point me in the right direction if I'm doing something blatantly wrong here.
I have the dependency in my `package.json` as well:
```
"dependencies": {
    "cloudworker-proxy": "^1.16.1"
  }
```

And here's my `wrangler.toml` for good measure:
```
name = "cloudflare-webhook"
type = "webpack"
account_id = "xxx"
workers_dev = true
route = ""
zone_id = ""
```

No errors reported by the worker. I'm kinda new to Cloudflare workers, so bear with me. Also - it was unclear to me from the documentation, but can you specify multiple `hosts` in the config? IE, can you forward one request to the worker to multiple endpoints?

**Comment by markusahlstrand** [comment]
Hi @crockk !

I've been using this in the past to split webhook-requests to multiple endpoints with a config like this:
 
```
{
    "path": "/webhook",
    "method": ["POST"],
    "host": "example.com",
    "handlerName": "split",
    "options": {
      "host": "split.example.com"
    }
  },
```

After the split handler a separate request will be passed down the handler-pipeline which is identical but with a different hostname. This split request is fire-and-forget so no return values will be passed back to the client.

To better see how it works you can add a logger handler after the split handler and it should generate two log entries for each inbound request.

To make an outbound call for the split request you need to add for instance a loadBalancer handler that matches the split hostname like this:
 ```
 {
    "host": "split.example.com",
    "path": "/:file*",
    "method": ["POST"],
    "handlerName": "loadbalancer",
    "options": {
      "sources": [
        {
          "url": "https://some.other.website.com/webhooks/doStuff"
        }
      ]
    }
  },
```

I guess it should work to have the request fall through to the origin as well, but I haven't tested it :) I think one issue with current configuration you're using is that the host parameter only handles host and not any path (no sure about port.. might work).

Let me know if you're still having issues.

**Comment by crockk** [comment]
Thanks for the quick response! 

After further deliberation, I don't think I had the use case right for the Split handler. As I understand, it splits an incoming request into two, redirecting it to another specified host. What I needed to do was receive an incoming request, validate it, and if valid, arbitrarily send out `n` requests to `n` endpoints (greater than 2). I came up with my own custom configuration to achieve this.
